pkg/custom_detectors/custom_detectors.go

```diff
@@ -109,6 +110,10 @@ func (c *CustomRegexWebhook) FromData(ctx context.Context, verify bool, data []b
 	return results, nil
 }
 
+func (c *CustomRegexWebhook) IsFalsePositive(_ detectors.Result) bool {
```

Comment:
**Nit:** If all parameters are ignored, you can omit `_`. You can even omit `c` in the pointer receiver.

```go
func (*CustomRegexWebhook) IsFalsePositive(detectors.Result) bool
```